apps/web/src/services/dashboards/shared/dashboard-widget-input-form/DashboardWidgetInputForm.vue

```diff
@@ -24,6 +24,7 @@
         >
             <p-text-button icon-left="ic_refresh"
                            style-type="highlight"
+                           :disabled="!isFormDataChanged"
                            class="return-to-initial-settings-button"
                            @click="handleReturnToInitialSettings"
             >
```

Comment:
description: disable button when options are not changed

![스크린샷 2023-08-08 오후 1 39 58](https://github.com/cloudforet-io/console/assets/18563857/bf0fc3e5-fe11-481a-ba1b-3142b7b49da0)